js/index.js

```diff
@@ -0,0 +1,36 @@
+// Psuedo code
+// 1 - doc ready
+// 2 - make submit clickable via .submit
+// 3 - read the value entered into the field; store as var
+// 4 - make change all vals to lowercase
+// 5 - if-else to say if var is city name, then  render new background image
+
+
+$(document).ready(function() {
+	//console.log('got here');
+
+	$('#submit-btn').click(submit);
+
+
+	function submit (event) {
+		event.preventDefault();
+		var location = $('#city-type').val();
+		var location = location.toLowerCase().trim();
```

Comment:
Nice use of `.toLowerCase()` and `.trim()` 👍 . 

However, you shouldn't double declare the same variable.

Declaration is this:
```js
var city;
``` 
It basically just says, he program, this variable exists now.
Assignment is this:
```js
city = 'something'; // it could be anything on the right hand side of the equal sign
```
This says, hey city variable that I already declared, I want you to be this value.

You can do both declaration and assignment in one line like so:
```js
var city = 'something';
```
That both declares the variable (`var city`) and assigns it (`city = something;`).

Once a variable is declared and assigned, you can *reassign* it just like a normal assignment.

```js
var city = 'something';

// this is the REassignment
city = 'something else';
```

Reassignments are just like assignments. Its simply `variableName = valueYouWantSetToTheVariable`.

But note that when reassigning a variable, you do not put `var` in front of it again. `var` only happens once: the very first time you declare a variable.

Also, for what its worth, you must always *declare* a variable before *assigning* a variable.